src/print/print.ts

```diff
@@ -143,14 +143,11 @@ function getPrintSettings(options: PrintOptions): string[] {
     printSettings.push(`bin=${bin}`);
   }
 
+  // paper size is represented by the drop down from your printer preference.
+  // For example if you need 4"x3" labels, you would set papersize to
+  // { paperSize: "4.00\"x3.00\"(102mm x 76mm)" }
   if (paperSize) {
-    if (validPaperSizes.includes(paperSize)) {
```

Comment:
please delete `validPaperSizes` array, since it's not used anymore. 
and remove valid sizes from readme too https://github.com/artiebits/pdf-to-printer/blob/master/README.md?plain=1#L72

Reply:
the rest looks fine to me. thanks again for contributing!

Reply:
np @artiebits  - removed the unused validPaperSizes array and also updated the readme to give a little more guidance on it. Let me know what you think. Thanks.